windows/tag_converters/converter_base.py

```diff
@@ -38,6 +38,7 @@ def setup_window(self, *args, **kwargs):
         except Exception:
             pass
 
+        # do path_replace to make sure the path works on linux
         tags_dir = path_replace(Path(curr_dir, 'tags'), 'tags', 'tags')
```

Comment:
I've invented a solution that you should merge from posix compat in supyr.
```suggestion
         tags_dir = Path(path_split(curr_dir, "tags", after=True))
```